src/brs/db/sql/SqlTradeStore.java

```diff
@@ -49,7 +52,10 @@ public Collection<Trade> getAllTrades(int from, int to) {
 
   @Override
   public Collection<Trade> getAssetTrades(long assetId, int from, int to) {
-    return tradeTable.getManyBy(TRADE.ASSET_ID.eq(assetId), from, to);
+    List<SortField<?>> sort = new ArrayList<>();
```

Comment:
@jjos2372 seems that the issue persists

![image](https://user-images.githubusercontent.com/3920663/203972698-2242e055-dbb3-4730-9c60-f75b7170b000.png)


![image](https://user-images.githubusercontent.com/3920663/203972506-d330c40d-7964-4123-8e42-905f5bd05c6e.png)